### PR TITLE
Fixed a bunch of typos

### PR DESCRIPTION
**Pull request by PaulMcClernan**

### CODE REVIEW

**Review by PaulMcClernan — COMMENTED** [review]
Looks good to me

File: docs/dictionary/command/edit.lcdoc [hunk 1]
```diff
@@ -25,7 +25,7 @@ edit the script of player "Show Me"
 
 Parameters:
 object:
-Any object reference. Usint the optional at syntax will pass the
+Any object reference. Use the optional at syntax will pass the
```
Comment (livecodepanos) [inline docs/dictionary/command/edit.lcdoc]: This should be `Using`

**Comment by livecodepanos** [comment]
Hello @PaulMcClernan 

This PR is almost ready. Some things to change:

Would it be possible to revert the last commit (since it is not just a correction of a typo as the other commits), and submit a new PR (separate) with this change?

Also, regarding the change itself, you should use `>*Note:*` instead of just `Note`, just after the `Description` part. For example see the description and the note of the `OnSave` handler:
```
Description:
Handler the <OnSave> message to execute code when a stack containing the widget is saved.
>*Note:* Access to most script object operations is not allowed whilst an <OnSave>
handler is running.
```

You need to do something similar in the new PR. 
 
Finally, the commit msg in the new PR should *just* say:

`Add note that the 'current' form of TheClickButton has not yet been implemented`

Thanks :)

**Comment by livecodepanos** [comment]
Also, for the new PR make sure you branch off (and submit the PR against) `develop-9.6`

**Review by PaulMcClernan — COMMENTED** [review]
Refixified

**Comment by PaulMcClernan** [comment]
Git is such a p.i.a. 
Not sure how to remove a commit from a PR (using the web interface).
I guess I'll close this out, and resubmit another PR

**Pull request closed without merging**